src/Symfony/Component/PasswordHasher/Exception/LogicException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PasswordHasher\Exception;
+
+/**
+ * @author Robin Chalas <robin.chalas@gmail.com>
+*/
+class LogicException extends \LogicException implements ExceptionInterface
```

Comment:
The component uses a mixture of `\LogicException` and this logic exception. Personally, I'm always using `\LogicException` as I believe only runtime exceptions should be catched (and thus need to be identified using the custom `ExceptionInterface`).

Whatever we choose, we should use it consistently in the component :)